pkg/operator/controller/controller.go

```diff
@@ -172,9 +152,17 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	ingresses := &ingressv1alpha1.ClusterIngressList{}
 	err = r.Client.List(context.TODO(), &client.ListOptions{Namespace: r.Namespace}, ingresses)
 	if err != nil {
-		return reconcile.Result{}, fmt.Errorf("failed to list clusteringresses in namespace %s: %v", r.Namespace, err)
+		errs = append(errs, fmt.Errorf("failed to list clusteringresses in namespace %s: %v", r.Namespace, err))
+	} else if caSecret != nil {
+		if err := r.ensureRouterCAConfigMap(caSecret, ingresses.Items); err != nil {
+			errs = append(errs, fmt.Errorf("failed to ensure router ca config map: %v", err))
```

Comment:
What do you think about moving `ensureRouterCAConfigMap` up with `ensureRouterCACertificateSecret` (on which it depends)? Something like the following:
```go
	caSecret, err := r.ensureRouterCACertificateSecret()
	if err != nil {
		// ...
	} else {
		ingresses := &ingressv1alpha1.ClusterIngressList{}
		if err := r.Client.List(context.TODO(), &client.ListOptions{Namespace: r.Namespace}, ingresses); err != nil {
			// ...
		} else if err := r.ensureRouterCAConfigMap(caSecret, ingresses.Items); err != nil {
			// ...
```

Reply:
fixed